lib/src/over_react_redux/devtools/middleware.dart

```diff
@@ -37,12 +37,12 @@ class _ReduxDevToolsExtensionConnection {
 external _ReduxDevToolsExtensionConnection reduxExtConnect([dynamic options]);
 
 class _OverReactReduxDevToolsMiddleware extends MiddlewareClass {
-  Store _store;
-  _ReduxDevToolsExtensionConnection devToolsExt;
+  Store? _store;
+  _ReduxDevToolsExtensionConnection? devToolsExt;
   final Logger log = Logger('OverReactReduxDevToolsMiddleware');
 
   _OverReactReduxDevToolsMiddleware([Map<String, dynamic> options = const {}]) {
-    var windowConsole = getProperty(window, 'console');
+    var windowConsole = getProperty(window, 'console') as Object;
```

Comment:
Any reason not to type this as `Console`?

Reply:
Since we're interacting with this as a JS object using `callMethod`, there's not an advantage to typing it as the Dart `Console` type.